test/actions/game.actions.ts

```diff
@@ -68,58 +68,97 @@ describe('actions', () => {
-                player: 'alan', 
+                player: 'alan',
                 bid: 0,
                 pass: true
             });
         });
     });
 
-    xdescribe('registerPlayer', () => {
-        it('', () => {
-            // @TODO write test to registerPlayer action
+    describe('registerPlayer', () => {
+        it('add new player to board', () => {
+            // assign
+            const currentState = this.state;
+            const playerId = "dżesika";
+            // act
+            const nextState = gameReducer(currentState, registerPlayer(playerId));
+            //assert
+            should(nextState.players.length).be.equal(4);
+            should(nextState.players[3].id).be.equal(playerId);
+
         });
     });
 
-    xdescribe('setDeck', () => {
-        it('', () => {
-            // @TODO write test to setDeck action
+    describe('setDeck', () => {
+        it('set game deck', () => {
+            // assign
+            const currentState = this.state;
+            const deck = currentState.deck
+            deck.push(createCard('J♥'))
+            // act
+            const nextState = gameReducer(currentState, setDeck(deck));
+            //assert
+            should(nextState.deck).be.deepEqual(deck);
+            should(nextState.deck[3]).be.deepEqual({ rank: 'J', suit: '♥' });
         });
-    }); 
+    });
+
+    describe('shareStock', () => {
+        it('share card', () => {
+            // assign
+            const currentState = this.state;
+            currentState.cards['pic'] = [createCard('10♥'), createCard('10♦')];
+            currentState.cards['adam'] = [];
+            // act
+            const nextState = gameReducer(currentState, shareStock(currentState.cards.pic[0], 'adam'));
+            //assert
+            should(nextState.cards['pic'].length).be.equal(1);
+            should(nextState.cards['pic'][0]).be.deepEqual({ rank: '10', suit: '♦' });
+            should(nextState.cards['adam'].length).be.equal(1);
+            should(nextState.cards['adam'][0]).be.deepEqual({ rank: '10', suit: '♥' });
 
-    xdescribe('shareStock', () => {
-        it('', () => {
-            // @TODO write test to shareStock action
         });
-    });    
-    
-    xdescribe('assignStock', () => {
-        it('', () => {
-            // @TODO write test to assignStock action
+    });
+
+    describe('assignStock', () => {
+        it('assign stock to a bid winner', () => {
+            // assign
+            const currentState = this.state;
+            currentState.cards.pic = [];
+            currentState.stock = [
+                createCard('10♥'),
+                createCard('10♥'),
+                createCard('J♦')
+            ];
+            // act
+            const nextState = gameReducer(currentState, assignStock());
+            //assert
+            should(nextState.cards['pic'].length).be.equal(3);
+            should(nextState.cards['pic']).be.deepEqual(currentState.stock);
```

Comment:
as above, why not just:
should(nextState.cards.pic).be.deepEqual( [
   createCard('10♥'),
   createCard('10♥'),
   createCard('J♦')
] );

Worth checking that stock is empty as well